jans-cedarling/bindings/cedarling_python/print_documentation.py

```
@@ -81,15 +92,17 @@ def print_doc(type_value: any, module_name: str | None = None):
     message = "\n".join(message_lines)
 
     if module_name is not None:
-        message = message.replace(type_value.__name__, "{}.{}".format(
-            module_name, type_value.__name__))
+        message = message.replace(
+            type_value.__name__, "{}.{}".format(module_name, type_value.__name__)
```

Ruff (0.15.20)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)